test/behavior/sizeof_and_typeof.zig

```diff
@@ -292,3 +292,13 @@ test "@sizeOf optional of previously unresolved union" {
     const Node = union { a: usize };
     try expect(@sizeOf(?Node) == @sizeOf(Node) + @alignOf(Node));
 }
+
+test "reified unions with zero-size payload fields" {
+    comptime {
+        // https://github.com/ziglang/zig/issues/8277
```

Comment:
Why is having the link relevant for the issue? `git log --grep=8277` or `git blame` should also work.

Reply:
Ah yes, no TODO is implied or needed so the link can go. Thanks.